pkg/controller/route/route_controller.go

```diff
@@ -182,12 +183,12 @@ func (r *ReconcileRoute) Reconcile(request reconcile.Request) (reconcile.Result,
 				if aerr, ok := ierr.(awserr.Error); ok {
 					switch aerr.Code() {
 					default:
-						fmt.Println(aerr.Error())
+						//fmt.Println(aerr.Error())
```

Comment:
but_why.gif

Reply:
I had received some feedback a while back on not having anything print to the screen- I had commented them out, figuring I'd setup logging when I go back and refactor. I did go ahead and change those to log the events in the resource for this pull request.